### PR TITLE
Add Upgrade guide

### PR DESCRIPTION
**Pull request by binbashblog**
Added upgrade guide

Added it to readme.md

JQJ1GanDU3c5RZwNjBXk68wFdxEJKLwWZU

### CODE REVIEW

File: guides/Masternode_Upgrade_Cold_Hot_Linux.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+shekel-cli masternode status
+```
+
+If you see status `Masternode successfully started`, you've done it, congratulations. Go hug someone now :)
+It will take a few hours until the first rewards start coming in.
+
+Instead, if you get status `Masternode not found in the list of available masternodes`, you need a bit more patience. Distributed systems take a bit of time to reach consensus. Restarting the wallets and retrying the start has been reported to help by community members. This is how you restart the Linux wallet from the CLI:
+```
+shekel-cli stop
+# wait 30 seconds or so for the wallet to gracefully stop and then start it again
```
Comment (ghost) [inline guides/Masternode_Upgrade_Cold_Hot_Linux.md]: I've seen this take 1-2 minutes. Let's make it 60 seconds.

Quick way to check if it has stopped:
```
ps aux | grep shekeld
```

Comment (binbashblog) [reply guides/Masternode_Upgrade_Cold_Hot_Linux.md]: Yeah, good idea

**Comment by binbashblog** [comment]
I've made the suggestions, thanks

File: guides/Masternode_Upgrade_Cold_Hot_Linux.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
+## SHEKEL Cold + Hot wallet MasterNode upgrade guide
+
+> This is a community contributed guide. Feel free to suggest improvements via Issues or opening Pull Requests. Thank you!
+
+
+**!!! This guide is for upgrading an existing SHEKEL COLD & Hot wallet MasterNode using the new Shekel ZeroCoin wallet and chain !!!**
+
+### Requirements
+* Existing version of Shekel-qt on Windows v1.2.0.0 or higher
```
Comment (ghost) [inline guides/Masternode_Upgrade_Cold_Hot_Linux.md:9]: 1.3.0.0 here?

**Review by ghost — APPROVED** [review]
Thank you for the contribution!

File: guides/Masternode_Upgrade_Cold_Hot_Linux.md (hunk 2 above)
Comment (binbashblog) [reply guides/Masternode_Upgrade_Cold_Hot_Linux.md:9]: It's saying you need the black 1.2.1.0 wallet and not the old white one as a requirement to upgrade from

**Pull request merged**